src/cmake/externalpackages.cmake

```diff
@@ -174,6 +174,10 @@ checked_find_package (OpenJPEG VERSION_MIN 2.0
 # Note: Recent OpenJPEG versions have exported cmake configs, but we don't
 # find them reliable at all, so we stick to our FindOpenJPEG.cmake module.
 
+checked_find_package (OpenJPH VERSION_MIN 0.21
+                      RECOMMEND_MIN 0.21
+                      RECOMMEND_MIN_REASON "for NLT support")
```

Comment:
This is fine, but note that the RECOMMENDED stuff only has an effect if it specifies a version that's newer than the enforced VERSION_MIN. For example, if RECOMMENDED_MIN was 0.22, and 0.21 was found, it would allow it and proceed, but print a warning that it was older than recommended and say the reason.

Can I just verify with you, @richardssam, that these are the right version numbers you intended?

Reply:
Fixed.